tests/tracking/test_fluent.py

```diff
@@ -681,3 +682,42 @@ def test_delete_tag():
     with pytest.raises(MlflowException):
         mlflow.delete_tag("b")
     mlflow.end_run()
+
+
+@pytest.mark.large
+def test_universal_autolog_calls_specific_autologs_correctly(mocker):
```

Comment:
I think it would be cleaner to refactor this test case so that it's parameterized by the module name, MLflow autologging function, and supported configs via `@pytest.mark.parametrize`.